### DISCUSSION TITLE
What is the server from "start server" on help tab is for ?

### DISCUSSION THREAD

**Discussion by CoffiDev** [johnlindquist/kit#156] [Q&A]
I started the server from the help tab, and when I navigate to the "/" I get a response "Invalid JSON". What is the purpose of this server? Can I customize it in some way?

**Comment by johnlindquist** [comment]
The idea is to let you control Script Kit from other devices in your home. There's a quick video here:

https://twitter.com/johnlindquist/status/1377284862269267979

I haven't talked about it much because I haven't done a ton of testing with it. I also plan on exposing a "site" when you GET from that URL (which you tried and nothing happened) showing buttons to run scripts.

Feel free to try it out. Just POST a json object with a "script" property of the script you want to run. Does that make sense?

**Reply by CoffiDev** [reply]
Woow! It worked! 🤯 Pretty cool way to allow interacting with the scripts from other devices! This gives me a couple of cool ideas to play around with.

This leaves me with 2 more questions: 

1- how can I pass arguments to the script?

2- where can I find the code that is running this server? just out of curiosity 🤓

Thanks for the response

**Reply by johnlindquist** [reply]
Nice!

An "args" property should pass arguments.

Here's the code:

https://github.com/johnlindquist/kitapp/blob/beta/src/server.ts

**Reply by johnlindquist** [reply]
And please 🙏 let me know what you make and if there's anything I can do to make it better!
